pkg/api/api.go

```diff
@@ -106,7 +106,7 @@ func (hs *HttpServer) registerRoutes() {
 	r.Post("/api/snapshots/", bind(m.CreateDashboardSnapshotCommand{}), CreateDashboardSnapshot)
 	r.Get("/api/snapshot/shared-options/", GetSharingOptions)
 	r.Get("/api/snapshots/:key", GetDashboardSnapshot)
-	r.Get("/api/snapshots-delete/:key", reqEditorRole, DeleteDashboardSnapshot)
+	r.Get("/api/snapshots-delete/:key", reqEditorRole, wrap(DeleteDashboardSnapshot))
```

Comment:
mumble mumble. This should be a `DELETE` request. But we have to keep the current one for backward compatibility. 
If we dont fix this know we should create an issue for deprecating this endpoint and use `DELETE` in docs and frontend.

Reply:
There are a whole lot of things that be changed here. Why are they not in a group like all the other routes? Why is the url for getting a list of snapshots `/api/dashboard/snapshots` (and why is it not documented)?

I

Reply:
The delete route looks strange , have a vague recollection that the thought was that users could undo it by just loading a delete url , because it needs to work for remote and local , and if it’s remote the remote instance needed to be in the url